src/main/java/racingcar/InputHandler.java

```diff
@@ -0,0 +1,61 @@
+package racingcar;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputHandler {
+    public static List<String> getCarNames() {
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String input = Console.readLine();
+        validateCarNamesInput(input);
```

Comment:
```suggestion
    public static List<String> readCarNames() {
        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
        String input = Console.readLine();
        validateCarNamesInput(input);
```

더 명시적으로 read라는 이름을 사용하면 어떨까요?
get은 뭔가 getter 느낌도 나서 저는 별로 선호하지 않는데 어떻게 생각하시나요?

Reply:
getter와 혼동 가능성 있다는 것에 동감합니다!
추천해 주신 네이밍 3주차 미션에 반영하도록 하겠습니다 👍